R/functions.R

```diff
@@ -486,6 +486,18 @@ table_to_markdown <- function(table, additional_spaces = 3, dot_to_space = TRUE)
   return(out)
 }
 
+
+#' Aggregate multiple numerical materiality exposures into a single, qualitative
+#
+# @param x a vector of numeric materialities
+aggregate_quantitative_to_qualitative_materiality <- function(x) {
+  cut(
+    sum(x),
+    breaks = c(0, 4.5, 9.5, 1000),
```

Comment:
They are defined so that 10 and above is "high", between 5 (incl.) and 10 (excl.) is medium. instead of 1000 I probably should use infinity (but up to at least 100 classes it makes no difference)

Reply:
Added the explanation from comment above to the docstring